locale/fi/LC_MESSAGES/django.po

```diff
@@ -3699,6 +3772,22 @@ msgstr "Opiskelijanumero"
 msgid "E-mail"
 msgstr "Sähköposti"
 
+#: userprofile/templates/userprofile/support.html
+msgid "%(brand_name)s Support "
+msgstr "%(brand_name)s-tuki "
+
+#: userprofile/templates/userprofile/support.html
+msgid "Hello"
+msgstr "Hei"
+
+#: userprofile/templates/userprofile/support.html
+msgid "User"
+msgstr "Käyttäjä"
+
+#: userprofile/templates/userprofile/support.html
+msgid "what can we\thelp with?"
```

Comment:
The tab character in the middle probably wasn't intentional. It can be changed in the HTML file and this po file is then generated again.